server/service/orbit.go

```diff
@@ -912,6 +912,13 @@ func (svc *Service) SaveHostScriptResult(ctx context.Context, result *fleet.Host
 			); err != nil {
 				return ctxerr.Wrap(ctx, err, "create activity for script execution request")
 			}
+
+			// lastly, queue a vitals refetch so we get a proper view of inventory from osquery
+			if activityStatus == "uninstalled" {
+				if err := svc.ds.UpdateHostRefetchRequested(ctx, host.ID, true); err != nil {
```

Comment:
Any chance we can do some new functionality to just refetch the queries we are interested in?
(Am guessing you are interested in the policy and software?)

E.g. a full refetch on my macOS device makes the host run 50 queries (policies, labels, software ingestion, etc.).
Maybe a-ok and we can patch if we see any issues.

Reply:
Perhaps a quick thing we can try in the case of load issues is only ask for a refetch if one hasn't been requested in the last few seconds 🤔

Reply:
> Guessing you're talking about potential back to back refreshes if we have some quick installs?

Yeah, that's what I meant 💯

Reply:
Yeah, apart from Fleet server load I was also worried about the host side (fleetd). We want to keep fleetd unnoticeable to end users. But again, I'm ok with merging and keeping an eye on this new behavior.